tests/model/test_grids.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+
+from cascade.model.grids import PriorGrid, AgeTimeGrid, _any_close
+from cascade.model.priors import GaussianPrior
+
+
+@pytest.fixture
+def age_time_grid():
+    return AgeTimeGrid.uniform(age_start=0, age_end=120, age_step=5, time_start=1990, time_end=2018, time_step=1)
+
+
+def test_PriorGrid__development_target():
+    # Make a grid using a standard structure. Other options might be
+    # a constructor which makes a grid that's denser at extreme ages.
+    grid = AgeTimeGrid.uniform(age_start=0, age_end=120, age_step=5, time_start=1990, time_end=2018, time_step=1)
```

Comment:
Would a denser grid also be created as a fixture?

Reply:
You could. For these particular tests the exact structure of the grid doesn't really matter.

Reply:
Ok, I was just noticing that you aren't using the existing fixture for this test.

Reply:
Oh, yeah. I left it there because this was my target while I was writing out the implementation and I wanted to keep it as a mini-example.